mapswipe_workers/mapswipe_workers/utils/tile_grouping_functions.py

```diff
@@ -318,55 +373,17 @@ def groups_intersect():
 
             if groups_intersect():
```

Comment:
Same comment as above: I would make this a pure function (pure meaning it has no side effect and all the data it needs is clearly passed through arguments).